OneCommunityGlobal/HighestGoodNetworkApp
# Raul pedro redesign reports page

# Description
(REPORTS COMPONENT - PRIORITIES HIGH) - **Continuation of [PR 641](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/pull/641)**

![image](https://user-images.githubusercontent.com/29555732/229851437-3840f570-a250-4831-9233-91b17571d34d.png)

## Mainly changes explained:
The reports page was redesigned. Now, it's possible to get data from teams and shows it with charts. It's possible to search teams by name etc.

## How to test:
1. git checkout raul_pedro_redesign_reports_page
2. npm install
3. npm run start:local
4. log in as admin.
5. Reports -> Teams -> Choose a team (the one with my name has data, so it's better for testing).
6. Test all features on the reports page (searching, selecting, etc.) and ensure it all works.

### Here is a video to help you:

https://user-images.githubusercontent.com/29555732/229852623-00fab088-047c-43cf-9947-cc890a49eee5.mp4
